client/src/assets/styles/global.scss

```diff
@@ -0,0 +1,5 @@
+// Typography
```

Comment:
Will this file contain all styles for the app? Saw something about importing styles in App.vue so curious as to how this will be extended for other components?

Reply:
Makes sense! We should leave a comment on this file to that effect.